main.tf

```diff
@@ -56,7 +56,14 @@ locals {
 
   labels = [for l in local.label_order : local.id_context[l] if length(local.id_context[l]) > 0]
 
-  id = lower(join(local.delimiter, local.labels))
+  full_id = lower(join(local.delimiter, local.labels))
```

Comment:
let's use `id_full` to be consistent with the other ones.

Reply:
or `id_original`

@Nuru @aknysh any thoughts?

Reply:
@osterman Updated 👍